### DISCUSSION TITLE
Q: 显示”url `/.netlify/functions/comment/ui/login` not found“ || Q: Display "url `/.netlify/functions/comment/ui/login` not found"

### DISCUSSION THREAD

**Discussion by DuceX** [walinejs/waline#2308] [Q&A] [answered]
### Describe the bug | 问题描述

部署完成后显示”url `/.netlify/functions/comment/ui/login` not found“，如何处理？

### Website URL | 问题网站

https://curious-biscuit-57708a.netlify.app/.netlify/functions/comment

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

After the deployment is completed, "url `/.netlify/functions/comment/ui/login` not found" is displayed. How to deal with it?

### Website URL | Problem website

https://curious-biscuit-57708a.netlify.app/.netlify/functions/comment

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is the data stored?

GitHub

**Comment by lizheming** [comment]
测了下，应该是 Netlify 的环境变量发生了些变化，导致环境判断有点问题，晚点我处理下

---
After testing, it seems that some changes have occurred in Netlify's environment variables, causing some problems in environmental judgment. I will deal with it later

**Comment by lizheming** [comment, marked as answer]
可以重新 fork 下试试哈，@waline/vercel@1.31.9 和新的模板已经适配了

---
You can fork it again and try it. @waline/vercel@1.31.9 and the new template have been adapted

**Comment by cloudshadow7** [comment]
### Describe the bug | 问题描述
类似问题，LeanCloud迁移到MongoDB，部署完成后显示”url `/.netlify/functions/comment` Not Found“，回滚到LeanCloud依旧 Not Found，请问如何处理？
### Website URL | 问题网站
https://waline7.netlify.app/.netlify/functions/comment
### Where your waline deploy? | 服务部署在哪里？
Netlify
### Where your comment data store? | 数据存储在哪里？
LeanCloud(https://leancloud.app)、MongoDB


---
### Describe the bug | Problem description 
Similar problem: LeanCloud is migrated to MongoDB. After the deployment is completed, "url `/.netlify/functions/comment` Not Found" is displayed. Rolling back to LeanCloud is still Not Found. How to deal with it? 
### Website URL | Problem website 
https://waline7.netlify.app/.netlify/functions/comment 
### Where your waline deploy? | Where is the service deployed? 
Netlify 
### Where your comment data store? | Where is the data stored? 
LeanCloud(https://leancloud.app)、MongoDB
